File: 0448/solution.java
import java.util.List;
import java.util.ArrayList;
import java.util.Set;
import java.util.HashSet;

class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        Set<Integer> diss = new HashSet<>();
        for(int i = 1; i<= nums.length; i++) {
            diss.add(i);
        }
        for(int num: nums) {
            diss.remove(num);
        }
        return new ArrayList<>(diss);
    }
}

File: 0448/solution.ts
function findDisappearedNumbers(nums: number[]): number[] {
  for (let i = 0; i < nums.length; i++) {
    const j = Math.abs(nums[i]) - 1;
    nums[j] = -Math.abs(nums[j]);
  }

  const res: number[] = [];
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] > 0) {
      res.push(i + 1);
    }
  }
  return res;
}
